backend/de.metas.business/src/main/java/de/metas/order/IOrderDAO.java

```diff
@@ -115,7 +115,7 @@ default I_C_OrderLine getOrderLineById(@NonNull final OrderAndLineId orderAndLin
 
 	/**
 	 * @param order
-	 * @param line
+	 * @param lineNo
 	 * @param clazz
 	 */
 	<T extends org.compiere.model.I_C_OrderLine> T retrieveOrderLine(I_C_Order order, int lineNo, Class<T> clazz);
```

Comment:
nothing to do know, but if you come across useless javadoc link this, you can in future remove it altogether